lib/topological_inventory/orchestrator/deployment_config.rb

```diff
@@ -95,7 +95,10 @@ def name
       def container_env_values
         [
           { :name => "INGRESS_API", :value => "http://#{ENV["TOPOLOGICAL_INVENTORY_INGRESS_API_SERVICE_HOST"]}:#{ENV["TOPOLOGICAL_INVENTORY_INGRESS_API_SERVICE_PORT"]}" },
-          { :name => "CONFIG", :value => 'custom'}
+          { :name => "CONFIG", :value => 'custom'},
+          { :name => "CLOUD_WATCH_LOG_GROUP", :value => "platform-dev" },
```

Comment:
@carbonin is the "CLOUD_WATCH_LOG_GROUP" set to different values based on Dev/CI/QA/Prod (if yes, how)? All topological inv. templates in e2e-deploy have "platform-dev" but you mentioned "platform"

Reply:
..solved